src/prefect/records/store.py

```diff
@@ -0,0 +1,12 @@
+from pydantic.v1 import BaseModel
+
+
+class RecordStore(BaseModel):
```

Comment:
Should this inherit from `BaseModel`? I think this makes sense if we expect to serialize `RecordStore` objects or expect to need validation when creating `RecordStore` objects, but if we don't need what `BaseModel` offers, then it might be best to have `RecordStore` be a plain Python ABC.

Reply:
Definitely doesn't need to, game to change that for sure